src/partition.rs

```diff
@@ -340,7 +340,9 @@ pub enum PredictionMode {
   PAETH_PRED,
   UV_CFL_PRED,
   NEARESTMV,
-  NEARMV,
+  NEAR0MV,
```

Comment:
Wow! Am I seeing AV2 before anyone else? :)
I thought all these prediction modes are strictly follows the av1 spec? So, no variants of NEARMV is possible?
Because the integer enum id for each mode is directly fed to symbol writing, so unless you parse again these new pred modes GLOBALMV will have three values shifted down?

Reply:
The values assigned to each mode are somewhat arbitrary. There is already a mismatch with libaom caused by UV_CFL_PRED I think. The generated bitstream still complies with the av1 spec.

Reply:
Unfortunately currently write_intra_mode_kf actually uses the numeric values. This only applies to the first modes (intra ones) of course, not to any past UV_CFL_PRED, and we could actually make a mapping function or something if we wanted.